### PR TITLE
ESLint script at `npm run lint`

### PR DESCRIPTION
**Pull request by Saatvik-droid**
For Issue #17

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,6 +29,7 @@
     "chai": "^4.3.4",
     "eslint": "^7.32.0",
     "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-config-google": "^0.14.0",
```
Comment (YashKumarVerma) [inline package.json:32]: not needed, we are using airbnb style

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Saatvik-droid) [reply package.json]: That should be fixed now

**Comment by Saatvik-droid** [comment]
I have no idea what these package-lock.json and package.json conflicts have come from

**Comment by divyanshu-parihar** [comment]
@Saatvik-droid  what happened after doing "git pull"?

**Comment by Saatvik-droid** [comment]
> 
> 
> @Saatvik-droid what happened after doing "git pull"?

Do you want me to run `git pull`? I have just cloned the repo, run npm i and pushed my changes.

**Comment by YashKumarVerma** [comment]
Hey there, conflicts come when your copy of the repository is outdated from what we have on server. To fix this,  you need to pull the changes and then resolve the merge conflicts.

**Comment by Saatvik-droid** [comment]
I know that, but when I try to install these new deps the lock file changes to use you use version 2 #23 and therefore the conflicts

**Pull request closed without merging**